tox.ini

```diff
@@ -38,6 +38,12 @@ passenv =
     TEST_BIGDATA
 
 commands =
+    # The following fix for C-extensions was copied from
+    # https://github.com/spacetelescope/synphot_refactor/blob/
+    # d4aa97f47a2b83e4b255f42087be446f94906354/tox.ini#L56-L58
+    # FIXME: Not sure why need this for tox to see the C-ext
```

Comment:
If you find a solution, please circle back to spacetelescope/synphot_refactor#295 . Thanks!

Reply:
I think I found one for `jwst` but it does not work for `synphot_refactor` yet (one failing test). I'll make another PR for `jwst` pipeline first and and then if you like it - you can try it on `synphot_refactor`.

The biggest issue that's killing this idea for `synphot_refactor` is your import of `_astropy_init` (due to affiliated package status).

Reply:
Why would `_astropy_init` break `tox`? 🤔